modules/admin_manual/pages/maintenance/upgrading/upgrade_php.adoc

```diff
@@ -11,11 +11,11 @@ And if you're on a version of PHP older than {minimum-php-version} you *must* up
 This guide steps you through upgrading your installation of PHP to one of the supported PHP versions ({supported-php-versions}) on Red Hat or CentOS 7.
 
 * xref:upgrade-php-to-version-5-6[Upgrade PHP to version 5.6]
-* xref:upgrade-php-to-version-7-0[Upgrade PHP to version 7]
+* xref:upgrade-php-to-version-7-0[Upgrade PHP to version {minimum-php-version}]
 
 == Upgrade PHP to version 5.6
 
-TIP: You should really be upgrading to PHP 7, as version 5.6 is https://secure.php.net/supported-versions.php[no longer actively supported], and security support ends on 31 Dec, 2018.
+TIP: You should really be upgrading to PHP {minimum-php-version}, as version 5.6 is https://secure.php.net/supported-versions.php[no longer actively supported], and security support ends on 31 Dec, 2018.
```

Comment:
Probably this whole section should be removed in 10.3 branch, because 10.3 does not support PHP 5.6 anyway.
It looks like that is all done a different way in 10.4 and master branch.
Is it reasonable to backport the "new" way from there? - code like:
```
:from-version: 5.6
:to-version: 7.1
:to-pkg-version: 71

include::{partialsdir}/maintenance/upgrading/upgrade_steps.adoc[leveloffset=+1]
```

Or do we just "maually" delete the `Upgrade PHP to version 5.6` section here?

Reply:
I'll submit a separate PR to update this section.